components/HeroBlockTechnology/HeroBlockTechnology.scss

```diff
@@ -120,23 +120,21 @@
 
       @include tablet-xl {
         left: 0;
-        bottom: -507px;
-        width: 290px;
-        height: 985px;
+        top: 144px;
+        width: auto;
```

Comment:
It seems that it would be better to use 100% ( and this property is used by default ) so you can remove this property